src/PrettierEditProvider.ts

```diff
@@ -44,6 +47,61 @@ const PARSER_SINCE = {
 function parserExists(parser: ParserOption, prettier: Prettier) {
     return semver.gte(prettier.version, PARSER_SINCE[parser]);
 }
+
+let statusbar: StatusBarItem = undefined;
```

Comment:
We should create it directly, not on first run. It gives the information: Prettier is up !